radicle-cli/src/commands/comment.rs

```diff
@@ -83,7 +83,8 @@ impl Args for Options {
 
         Ok((
             Options {
-                id: id.ok_or_else(|| anyhow!("an issue id to comment on must be provided"))?,
+                id: id
+                    .ok_or_else(|| anyhow!("an issue/patch id to comment on must be provided"))?,
```

Comment:
this id can be both issue and patch, correct?

Reply:
Yes

Reply:
```suggestion
                    .ok_or_else(|| anyhow!("an issue or  patch to comment on must be provided"))?,
```